ngmix/em_nb.py

```diff
@@ -113,8 +113,8 @@ def do_scratch_sums(pixel, gmix, sums):
         chi2 = \
             gauss['dcc']*v2 + gauss['drr']*u2 - 2.0*gauss['drc']*uv
 
-        if chi2 < 25.0 and chi2 >= 0.0:
-            tsums['gi'] = gauss['pnorm']*expd( -0.5*chi2 )
+        if chi2 < FASTEX_MAX_CHI2 and chi2 >= 0.0:
+            tsums['gi'] = gauss['pnorm']*exp5(-0.5*chi2)
```

Comment:
Should we hide this def behind a simple global name? 

```
from .fastexp import fexp, FASTEXP_MAX_CHI2
...
```

where `fexp` is

```
if True:
    fexp = exp5
elif False:
    fexp = exp3
```

etc.

Reply:
I actually like seeing the exp5 there for now, but I agree we should maybe rename later